site/gatsby-site/src/graphql/checklists.js

```diff
@@ -1,4 +1,5 @@
-import gql from 'graphql-tag';
+//import gql from 'graphql-tag';
+import { gql } from '../../server/generated';
 
 const allChecklistFields = `
```

Comment:
This took me a while to figure out! 

graphql codegen didn't like using a variable to create the query, so it did not validate the queries in this file.

(I think it is because it performs only static analysis.)